managed/services/management/agent.go

```diff
@@ -137,15 +137,11 @@ func (s *AgentService) agentToAPI(agent *models.Agent) (*agentv1beta1.UniversalA
 	ua := &agentv1beta1.UniversalAgent{
 		AgentId:                        agent.AgentID,
 		AgentType:                      string(agent.AgentType),
-		AwsAccessKey:                   pointer.GetString(agent.AWSAccessKey),
 		CreatedAt:                      timestamppb.New(agent.CreatedAt),
 		CustomLabels:                   labels,
 		Disabled:                       agent.Disabled,
 		DisabledCollectors:             agent.DisabledCollectors,
 		IsConnected:                    s.r.IsConnected(agent.AgentID),
-		IsAgentPasswordSet:             agent.AgentPassword != nil,
```

Comment:
@JiriCtvrtka We need to create a task to update frontend accordingly, since Inventory now uses those keys.

Reply:
@JiriCtvrtka you said that we don't encrypt values if they are nil, so I think these 3 ifs should work as it used to, right?